solutions/scripts/avn-power-services.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+# jq needs to be installed https://stedolan.github.io/jq/
```

Comment:
Would this (or a smaller version of it?) be useful in the USAGE output? For those folks like me who just download and run things :)

Reply:
I like this idea! A bit like Git that says "no no but you might have wanted to write this instead?" ;)